addons/mail/static/src/discuss/call/common/call_preview.js

```diff
@@ -140,36 +147,62 @@ export class CallPreview extends Component {
             name: ({ action }) => (action.isActive ? _t("Unmute") : _t("Mute")),
             onSelected: () => this.toggleMic(),
         };
-        return [
-            [
+        const videoBlurAction = {
+            condition: () => this.state.videoStream !== null,
+            icon: () => "fa fa-fw fa-photo",
+            isActive: ({ store }) => store.settings.useBlur,
+            name: ({ action }) =>
+                action.isActive ? _t("Disable background blur") : _t("Enable background blur"),
+            onSelected: ({ store }) => {
+                store.settings.useBlur = !store.settings.useBlur;
+            },
+            tags: ({ action }) => (action.isActive ? [ACTION_TAGS.SUCCESS] : []),
+        };
+        const callAudioActions = [
+            new Action({
+                id: "toggle-microphone",
+                owner: this,
+                definition: muteActionUpdated,
+                store: this.store,
+            }),
+        ];
+        const callVideoActions = [
+            new Action({
+                id: "toggle-camera",
+                owner: this,
+                definition: cameraOnActionUpdated,
+                store: this.store,
+            }),
+        ];
+        if (this.props.settingsOutOfActionList === true) {
```

Comment:
Just checking truthy is enough. By default it's `undefined` which is falsy so it's ok. And if someone wants to pass `1` I think it's fair to accept as equivalent to `true`.